pkg/apis/stork/v1alpha1/volumeexport.go

```diff
@@ -0,0 +1,116 @@
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// VolumeExportResourceName is name for the VolumeExport resource.
+	VolumeExportResourceName = "volumeexport"
+	// VolumeExportResourcePlural is the name for list of VolumeExport resources.
+	VolumeExportResourcePlural = "volumeexports"
+)
+
+// ExportType defines a method of achieving data transfer.
+type ExportType string
+
+const (
+	// ExportRsync means that data will be copied between two PVCs directly.
+	// Rsync supports both local and remote file copy.
+	ExportRsync ExportType = "rsync"
+	// ExportExtent means that some tool/service will handle data transfer.
```

Comment:
Lets only keep rsync for now. We will add the other methods once we have an implementation.

Reply:
Updated.